test/functorch/test_vmap_registrations.py

```diff
@@ -67,14 +68,331 @@
     "aten::unfold_copy",
 }
 
+xfail_not_implemented = {
+    "aten::absolute_",
+    "aten::affine_grid_generator_backward",
+    "aten::align_as",
+    "aten::align_tensors",
+    "aten::align_to",
+    "aten::align_to.ellipsis_idx",
+    "aten::alpha_dropout",
+    "aten::alpha_dropout_",
+    "aten::arccos_",
+    "aten::arccosh_",
+    "aten::arcsin_",
+    "aten::arcsinh_",
+    "aten::arctan2_",
+    "aten::arctan_",
+    "aten::arctanh_",
+    "aten::argwhere",
+    "aten::bilinear",
+    "aten::bitwise_and_.Scalar",
+    "aten::bitwise_or_.Scalar",
+    "aten::bitwise_xor_.Scalar",
+    "aten::can_cast",
+    "aten::cat.names",
+    "aten::chain_matmul",
+    "aten::chalf",
+    "aten::choose_qparams_optimized",
+    "aten::clip_",
+    "aten::clip_.Tensor",
+    "aten::coalesce",
+    "aten::column_stack",
+    "aten::concat.names",
+    "aten::concatenate.names",
+    "aten::conj",
+    "aten::conv_tbc_backward",
+    "aten::ctc_loss.IntList",
+    "aten::ctc_loss.Tensor",
+    "aten::cudnn_is_acceptable",
+    "aten::cummaxmin_backward",
+    "aten::cumprod_backward",
+    "aten::data",
+    "aten::diagflat",
+    "aten::divide.out_mode",
+    "aten::divide_.Scalar",
+    "aten::dropout",
+    "aten::dropout_",
+    "aten::embedding_bag",
+    "aten::embedding_bag.padding_idx",
+    "aten::embedding_sparse_backward",
+    "aten::fake_quantize_per_channel_affine",
+    "aten::fake_quantize_per_channel_affine_cachemask_backward",
+    "aten::fake_quantize_per_tensor_affine",
+    "aten::fake_quantize_per_tensor_affine.tensor_qparams",
+    "aten::fake_quantize_per_tensor_affine_cachemask_backward",
+    "aten::fbgemm_linear_fp16_weight",
+    "aten::fbgemm_linear_fp16_weight_fp32_activation",
+    "aten::fbgemm_linear_int8_weight",
+    "aten::fbgemm_linear_int8_weight_fp32_activation",
+    "aten::fbgemm_linear_quantize_weight",
+    "aten::fbgemm_pack_gemm_matrix_fp16",
+    "aten::fbgemm_pack_quantized_matrix",
+    "aten::fbgemm_pack_quantized_matrix.KN",
```

Comment:
Other things to filter out via regex to shrink the list some more:
- fbgemm (we likely don't see those)
- quantize / quant
- There are still some Dimname and names. It's OK if our filter is too aggressive
- The `is_` operations (those are usually implemented in a way that bypasses the dispatcher)
- The `sparse` operations (those are not supported by vmap)